### DISCUSSION TITLE
How To Make The Font Big

### DISCUSSION THREAD

**Discussion by Pgbgroup** [PartialVolume/shredos.x86_64#254] [Q&A] [answered]
How To Make The Font Big

**Comment by PartialVolume** [comment, marked as answer]
See [nwipes-font-size-is-too-small-how-to-double-the-size-of-the-text](https://github.com/PartialVolume/shredos.x86_64#nwipes-font-size-is-too-small-how-to-double-the-size-of-the-text). So you can double the size of the font by manually running a command.

However, currently you can't set this so it automatically doubles the size of the font at boot time. I can add a option to grub or nwipe.conf if anybody would like to be able to boot ShredOS with nwipe using double size font automatically.

**Comment by brutlern** [comment]
Yes, an automatic double size font on boot would be nice. One of my colleagues has eyesight issues and is annoyed about the default small font size. But this is more of a me problem.

**Reply by PartialVolume** [reply]
I'll see if I can add this feature for the next release. Maybe some or all of the following:

1. A option such as shredos_font=double that can be placed on the kernel command line so it boots up straight into double size font.
2. A option in config, so you can toggle font size.
3. A hot key like 'd' that will toggle the font size from the drive selection screen.

**Reply by brutlern** [reply]
Either option would be fine but I think the hotkey one is the most elegant solution, this way anyone can change on the fly depending on the monitor they are using or simple preference, but you can go with whichever option is simplest to implement from your side.

**Comment by PartialVolume** [comment]
Committed a script that toggles font size. https://github.com/PartialVolume/shredos.x86_64/commit/c21ac29363b6ea500aabdfac77e1865b097054ec

This will be activated by the d key in the drive selection window of nwipe.

**Comment by PartialVolume** [comment]
Changes commited to nwipe https://github.com/martijnvanbrummelen/nwipe/pull/589/commits/c024addd9382ac42c594723b5b0ff58d2bcb8731

**Comment by PartialVolume** [comment]
All done, this feature will be available in the next release of ShredOS.

**Comment by PartialVolume** [comment]
@brutlern if you wanted to test the toggle front size you can download a .img from the link in this post https://github.com/PartialVolume/shredos.x86_64/discussions/234#discussioncomment-10631315

Remember, to toggle the font size press the d key when on the drive selection screen. Toggling the font size using d won't work from other screens. So once you have doubled the size of the font it will remain double in size during the wipe, pressing d during a wipe won't toggle the size. I can add the ability to toggle front size from the wipe screen as well should you think that would be useful. So you can toggle the font between double and standard size by repeatedly pressing the d key.
